src/Symfony/Bridge/PhpUnit/Legacy/CoverageListenerTrait.php

```diff
@@ -13,7 +13,7 @@
 
 use PHPUnit\Framework\TestCase;
 use PHPUnit\Framework\Warning;
-use PHPUnit\Util\Test;
```

Comment:
We can keep the import.

Reply:
That was CS Fixer doings. Should I get it back in spite of it?

Reply:
OK, I got it back

Reply:
OK, I got it back

￼

Reply:
Yes, the CS Fixer removes it because you've replaced `Test` with its FQCN throughout this file. If you revert this as well, the CS Fixer won't complain anymore.